pandas_gbq/gbq.py

```diff
@@ -1014,6 +993,147 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
     return final_df
 
+def read_gbq(query, project_id=None, index_col=None, col_order=None, verbose=True,
```

Comment:
Looks like there was maybe a problem with the rebase. This is a duplicate definition of `read_gbq`.

Reply:
Wow, totally didn't notice since calling `read_gbq` uses the new one I wrote, which is further down. Kinda surprised python didn't complain, but maybe I shouldn't. Removed old one in latest commit, thanks for catching!